hack/deploy.sh

```diff
@@ -259,6 +235,11 @@ integrations() {
             --dockerconfigjson='"$QUAY__DOCKERCONFIGJSON"' \
             --token='"$QUAY__API_TOKEN"' --url='"$QUAY__URL"'
     fi
+    if [[ -n "${TAS:-}" ]]; then
+        tssc_cli integration trusted-artifact-signer --force \
+            --rekor-url='"$TAS__REKOR_URL"' \
+            --tuf-url='"$TAS__TUF_URL"'
```

Comment:
Any specific reason for the single and double quotes together? I think using a simple `"${TAS__REKOR_URL}"` should work as expected.

Reply:
Yes, it has to do with how the binary is then called, which can be either directly or through the image.
I agree it's a little bit to look at though, so if you have a better solution we can change it.